src/docfx/data/schemas/learningpath-error-mapping.json

```diff
@@ -17,11 +17,6 @@
         "severity": "error",
         "code": "path-modules-missing",
         "message": "Missing attribute: modules. At least one valid module is required for each learning path."
-      },
```

Comment:
add a comment or note to add it back once we support it well

Reply:
OK.